Reference:
    https://www.math-only-math.com/worksheet-on-word-problems-on-four-operations.html

<div class="alert alert-block alert-info">
<b>Tip:</b> En python podemos utilizar under score [ _ ] como separador de miles 
    en el código y no tiene efecto alguno, aparte de facilitar la lectura para el usuario
</div>

#### 1.Problem:
The population of a town is 198,568. Out of them 45,312 are men and 35,678 are women. Find the number of children in the town.

In [83]:
# Utilizamos el under score como separador de miles, que es inocuo a efectos de compilación, pero
# facilita la lectura al usuario.
total_population = 198_568
men = 45_312
woman = 35_678
child = total_population - (men+woman)
# Método de formateo numérico para verisiones  ≥ 3.6
print(f'Number of child {child:,}')

Number of child 117,578


<div class="alert alert-block alert-info">
<b>Tip:</b> Podemos utilizar varios métodos para formatear el resultados con separador de miles.<br><ul>
<li>El más sencillo es el de utilizar [<code>value:,</code>] que nos los coloca directamente.</li>
<li>El otro es el de utilizar la clase format, en el que le pasamos <code>format(value, ',d')</code></li></ul>
</div>

#### 2.Problem:
A shopkeeper has 2,425 boxes of 24 pencils each. How many pencils do all the boxes have in all?

In [59]:
number_of_boxes=2_425
pencils_per_box=24
total_number_of_pencils=number_of_boxes*pencils_per_box
print(f'Total number of pencils : {total_number_of_pencils:,}')

Total number of pencils : 58,200


#### 3.Problem:
Linda bought a coat for \\$2,265 and a saree for \\$2,150. She gave \\$5,000 to the shopkeeper. How much money did the shopkeeper return to her?

In [60]:
coat=2_265
saree=2_150
pay=5_000
change=pay-(coat+saree)
print(f'Total change: ${change:,}')

Total change: $585


#### 4.Problem:
The cost of 21 TV sets is $95,844 Find the cost of one TV set.

In [62]:
full_set = 21
full_set_cost=95_844
single_set_cost=full_set_cost/full_set
print(f'Single TV cost : ${single_set_cost:,}')

Single TV cost : $4,564.0


#### 5.Problem:
A factory produces 24,532 bulbs in a month. What is its annual production?

In [64]:
bulbs_per_month=24_532
annual_production=bulbs_per_month*12
print(f'Annual production {annual_production:,} bulbs.')


Annual production 294,384 bulbs.


#### 6.Problem:
There are 145,968 bags of sugar, 236,487 bags of wheat and some bags of rice in a godown. It the total number of bags in the godown is 450,000, find the number of bags of rice.

In [65]:
bags_sugar=145_968
bags_wheat=236_487
total_bags=450_000
bags_rice=total_bags-(bags_sugar+bags_wheat)
print(f'Total bags of rice: {bags_rice:,}')

Total bags of rice: 67,545


#### 7.Problem:
A factory manufactured 483,685 toys in three weeks. The production in first week was 146,345 toys and in second week 138,152 toys. Find the production in the third week.

In [69]:
total_production=483_685
first_week=146_345
second_week=138_152
third_week=total_production-(first_week+second_week)
print(f'Production on third week: {third_week:,}')

Production on third week: 199,188


#### 8.Problem:
The cost of a sofa set is $9,372. How much will 124 such sofa sets cost?

In [67]:
sofa_cost=9_372
number_of_sofas=124
total_cost=sofa_cost*number_of_sofas
print(f'Total cost of 124 sofas: ${total_cost:,}')

Total cost of 124 sofas: $1,162,128


#### 9.Problem:
There are 86 rooms in a school. 4,356 students study there. Equal number of students sits in each room?

In [74]:
total_rooms=86
total_students=4_356
students_per_room=total_students/total_rooms
print(f'The number of strudents per room is {round(students_per_room)}.')

The number of strudents per room is 51


<div class="alert alert-block alert-info">
<b>Tip:</b> Podemos utilizar la función round para redondear el resultado a la precisión requerida.<ol><code>round(value, precission)</code></ol>
</div>

#### 10.Problem:
1,575 students of a school want to go Agra by bus. If one bus can carry 75 students, how many buses are required to carry all the students?

In [76]:
total_students=1_575
bus_capacity=75
buses_needed=total_students/bus_capacity
print(f'We require {buses_needed:,} buses to carry everyone.')

We require 21.0 buses to carry everyone.


#### 11.Problem:
The cost of a radio set is $1,475. What is the cost of 35 such radio sets?

In [78]:
radio_set_cost=1_475
number_of_sets=35
total_cost=radio_set_cost*number_of_sets
print(f'The total cost of {number_of_sets} radio sets is ${total_cost:,}.')

The total cost of 35 radio sets is $51,625.


#### 12.Problem:
In an election, 52,496 people voted for Ron, 44,929 people for Jhon and 36,824 people for Mike in a town. If everyone voted in the town, what is the total number of voters?

In [81]:
ron_voters=52_496
jhon_voters=44_929
mike_voters=36_824
total_voters=ron_voters+jhon_voters+mike_voters
print(f'Total number of voters {total_voters:,}.')

Total number of voters 134,249.


#### 13.Problem:
Maria bought 96 toys priced equally for \\$12,960. The amount of \\$1,015 is still left with her. Find the cost of each toy and the amount she had.

In [82]:
number_of_toys=96
total_price=12_960
toy_price=total_price/number_of_toys
money_left=1_015
total_money=total_price+money_left
print(f'Each toy cost ${toy_price:,}. Maria had ${total_money:,}')

Each toy cost $135.0. Maria had $13,975


### Answers:
1. 117,578 children.
2. 58,200 pencils.
3. \\$585
4. \\$4,564
5. 294,384 bulbs.
6. 67,545 bags.
7. 199,188 toys.
8. \\$1,162,128
9. 51 students.
10. 21 buses.
11. \\$51,625
12. 134,249
13. \\$135, \\$13975